desc/examples/reactor_QA.py

```diff
@@ -0,0 +1,87 @@
+"""Example script for recreating the QA configuration of Landreman, Buller & Drevlak."""
```

Comment:
add more description, and emphasize that it is not the exact same as that paper (same for precise QA/QH)